docs/prover/cli/options.md

```diff
@@ -745,3 +745,13 @@ This option models bitwise operations exactly instead of using the default
 counterexample caused by incorrect modeling of bitwise operations, but can
 dramatically increase the time taken for verification.
 
+(-smt_groundQuantifiers)=
+#### `--settings -smt_groundQuantifiers=false`
+
+This option disables quantifier grounding.  See {ref}`grounding` for more
+information.
+
+```{todo}
```

Comment:
I'd reference to the doc above, it pretty much says when. In general it's when you get a counter example and it seems like it does not respect some quantified expression.